src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static List<User> userList;
+    private static User nullUser;
+    private static User userWithInvalidAge;
+    private static User userWithValidData;
+    private static User userWithValidDateSameLogin;
+    private static User userWithInvalidPassword;
+    private static User userWithValidData1;
+    private static User userWithValidData2;
+    private static User userWithValidDateSameLogin2;
+    private static User userWithNullAge;
+    private static User userWithNullLogin;
+    private static User userWithNullPassword;
```

Comment:
numbers in variable naming is a bad practice. I suggest cleaning this up and leaving only one user, which you will set up with valid attributes in BeforeEach, and then amend to your needs in each test like this:
![image](https://user-images.githubusercontent.com/49767049/227717102-dfb31f8d-c428-479e-acbe-82228f5db65c.png)